content/home/organizers.md

```diff
@@ -26,7 +26,9 @@ TBD
 ### Local Arrangements
 Arnaud Legrand (CNRS)  
 Camille Maumet (INRIA)  
-  
+Matthieu Simonin (INRIA)
```

Comment:
Add a double blank after each of the new names to make them appear on their own line (a quirk of markdown).

Reply:
Thankd @carlosmalt, that is now done ✔️